### ISSUE TITLE
PHP Crash after changing timezone

### ISSUE THREAD

**Issue by KurtzPT** [MrBrax/TwitchAutomator#13]
Using the updated docker image, after I changed the timezone + use the relative time option, PHP crashes with the following:

```
NOTICE: PHP message: PHP Notice:  Object of class DateTime could not be converted to int in /var/www/twitchautomator/public/index.php on line 112
NOTICE: PHP message: PHP Fatal error:  Uncaught Error: Call to a member function getTimestamp() on int in /var/www/twitchautomator/src/TwitchHelper.php:704
Stack trace:
#0 /var/www/twitchautomator/public/index.php(112): App\TwitchHelper::printHumanDate()
#1 /var/www/twitchautomator/cache/twig/3a/3af46ce2e928497da210547ab9e347c45509d5bbba0f5728fd511bd7932b6bda.php(201): {closure}()
#2 /var/www/twitchautomator/vendor/twig/twig/src/Template.php(396): __TwigTemplate_250c30b58123a49146e514f44416fc3fae41175b56139db280bca24fb51df348->doDisplay()
#3 /var/www/twitchautomator/vendor/twig/twig/src/Template.php(369): Twig\Template->displayWithErrorHandling()
#4 /var/www/twitchautomator/cache/twig/e8/e85cc74cbc4a38f6cc9567ca26c5caeae84eb679f1f7980293329d92e02eb25b.php(53): Twig\Template->display()
#5 /var/www/twitchautomator/vendor/twig/twig/src/Template.php(396): __TwigTemplate_5bd7e7e21523289160a18cd80b477ac9d40b9426df34c97f4ab1c1eac9674b44->doDisplay()
#6 /var/www/twitchautomator/vendor/twig/twig/src/Template.php(3...
```

**Comment by KurtzPT** [comment]
Another issue I encountered. Adding a password to the app will not work unless you have a functional https certificate.

> Type: RuntimeException
Code: 0
Message: Insecure use of middleware over HTTP denied by configuration.
File: /var/www/twitchautomator/vendor/tuupola/slim-basic-auth/src/HttpBasicAuthentication.php
Line: 148

According to [this](https://github.com/tuupola/slim-jwt-auth#security), if we pass "secure" as false, it will work. Can we add this setting to the ENV values?

**Comment by MrBrax** [comment]
Fixed in 731996fa68043fe87ddf8453726674a6b9c3203d
